app/models/bestuursorgaan.js

```diff
@@ -38,6 +38,28 @@ export default class BestuursorgaanModel extends Model {
   })
   bevat;
 
+  async classificatieUri() {
+    const bestuursorgaan = await this.isTijdsspecialisatieVan;
+    return bestuursorgaan
+      ? await bestuursorgaan.classificatie.uri
+      : await this.classificatie.uri;
+  }
+
+  // TODO decretale bestuursorganen should be fetched from database
+  // Bestuursorgaan and bestuursorgaan in de tijd are not the same. Use mandatendatabank schema for reference.
+  // go via isTijdsspecialisatieVan to go to the real bestuursorgaan.
```

Comment:
seems to be a personal comment, not something that should be in the repo

Reply:
i think it could be in this file, but all the way at the top as a general warning. This causes some major confusion when working with this type